### ISSUE TITLE
Advanced Content Type in 2.8 and relationship to it

### ISSUE THREAD

**Issue by rinoxstudio** [pods-framework/pods#6314]
### Description

In previous versions Advanced Content Type (ACT) had preset of fields, one of them was “title”. Now there is no any presets but you can’t create field named “title”, because of getting message, that “title” is reserved for this content type… In ACT created before 2.8 update field "title" exists and work well, but in new ACT's there is no way to use it.

Another problem is adding new ACT post with relationship field seems to be the next step of previous problem. For example I create field named my_field (multiple items, list view) related to ACT named my_act created after 2.8 with ability to add new ACT post. I click “Add new my_act” on field "my_field", fill data in modal, click save (“add new my_cpt” button in modal) and meet infinite rotating saving icon trying to save it. Nothing happens, so I close modal and as the result - my_act post was created but not added to field and cant be found by autocomplete before page reload.

### Version

2.8,5

### Testing Instructions

1. Activate advanced content types
2. Create new advanced post type item
3. Add to any post type relationship field (multiple, list view, allow add new) related to new advanced content type
4. Go to post editor where relationship field was added click "add new"
5. Fill in fields in modal. Click Create new {type name}.

### Site Health Information

```text
`
### wp-core ###

version: 5.8.2
site_language: ru_RU
user_language: ru_RU
timezone: +03:00
permalink: /%category%/%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: production
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/u0304268/data/www/fujimifoto.ru
wordpress_size: 66,22 МБ (69441523 bytes)
uploads_path: /var/www/u0304268/data/www/fujimifoto.ru/wp-content/uploads
uploads_size: 581,08 МБ (609306814 bytes)
themes_path: /var/www/u0304268/data/www/fujimifoto.ru/wp-content/themes
themes_size: 37,40 МБ (39217249 bytes)
plugins_path: /var/www/u0304268/data/www/fujimifoto.ru/wp-content/plugins
plugins_size: 50,09 МБ (52521864 bytes)
database_size: 10,27 МБ (10764288 bytes)
total_size: 745,06 МБ (781251738 bytes)

### wp-active-theme ###

name: BOYA (boya)
version: 1.0
author: Rinox Studio
author_website: http://rinox.by
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, post-thumbnails, menus
theme_path: /var/www/u0304268/data/www/fujimifoto.ru/wp-content/themes/boya
auto_update: Выключено

### wp-themes-inactive (1) ###

Twenty Twenty-One: version: 1.4, author: Команда WordPress, Автоматические обновления отключены

### wp-plugins-active (10) ###

Advanced Editor Tools (previously TinyMCE Advanced): version: 5.6.0, author: Automattic, Автоматические обновления отключены
Advanced Order: version: 2.0, author: c0ns0l3, Автоматические обновления включены
Breadcrumb NavXT: version: 6.6.0, author: John Havlik, Автоматические обновления включены
Contact Form 7: version: 5.5.2, author: Takayuki Miyoshi, Автоматические обновления включены
Cyr to Lat enhanced: version: 3.5, author: Sol, Sergey Biryukov, Nikolay Karev, Dmitri Gogelia, Автоматические обновления отключены
Pods - Custom Content Types and Fields: version: 2.8.5, author: Pods Framework Team, Автоматические обновления отключены
Pods SEO: version: 2.2.1, author: Pods Framework Team, Автоматические обновления отключены
Toolset Types: version: 2.2.9, author: OnTheGoSystems, Автоматические обновления отключены
Yoast Duplicate Post: version: 4.1.2, author: Enrico Battocchi & Team Yoast (latest version: 4.2), Автоматические обновления отключены
Yoast SEO: version: 17.4, author: Team Yoast (latest version: 17.6), Автоматические обновления отключены

### wp-plugins-inactive (1) ###

Password Protected: version: 2.5, author: Ben Huson, Автоматические обновления отключены

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-86 Q16 x86_64 2021-10-18 https://imagemagick.org
imagick_version: 3.4.4
file_uploads: File uploads is turned off
post_max_size: 256M
upload_max_filesize: 256M
max_effective_size: 256 МБ
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 251 ГБ
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 384
	imagick::RESOURCETYPE_MAP: 251 ГБ
	imagick::RESOURCETYPE_MEMORY: 126 ГБ
	imagick::RESOURCETYPE_THREAD: 1
imagemagick_file_formats: 3FR, 3G2, 3GP, AAI, AI, ART, ARW, AVI, AVS, BGR, BGRA, BGRO, BIE, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CR2, CR3, CRW, CUR, CUT, DATA, DCM, DCR, DCX, DDS, DFONT, DNG, DOT, DPX, DXT1, DXT5, EPDF, EPI, EPS, EPS2, EPS3, EPSF, EPSI, EPT, EPT2, EPT3, ERF, EXR, FAX, FILE, FITS, FRACTAL, FTP, FTS, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, GV, H, HALD, HDR, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, IIQ, INFO, INLINE, IPL, ISOBRL, ISOBRL6, J2C, J2K, JBG, JBIG, JNG, JNX, JP2, JPC, JPE, JPEG, JPG, JPM, JPS, JPT, JSON, K25, KDC, LABEL, M2V, M4V, MAC, MAGICK, MAP, MASK, MAT, MATTE, MEF, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MRW, MSL, MSVG, MTV, MVG, NEF, NRW, NULL, ORF, OTB, OTF, PAL, PALM, PAM, PANGO, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PDF, PDFA, PEF, PES, PFA, PFB, PFM, PGM, PGX, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, POCKETMOD, PPM, PREVIEW, PS, PS2, PS3, PSB, PSD, PTIF, PWP, RADIAL-GRADIENT, RAF, RAS, RAW, RGB, RGBA, RGBO, RGF, RLA, RLE, RMF, RW2, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, SR2, SRF, STEGANO, SUN, SVG, SVGZ, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WEBP, WMF, WMV, WMZ, WPG, X, X3F, XBM, XC, XCF, XPM, XPS, XV, XWD, YCbCr, YCbCrA, YUV
gd_version: bundled (2.1.0 compatible)
gd_formats: GIF, JPEG, PNG, WebP, BMP
ghostscript_version: 9.25

### wp-server ###

server_architecture: Linux 3.10.0-1160.25.1.el7.x86_64 x86_64
httpd_software: Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips mod_fcgid/2.3.9 Phusion_Passenger/5.3.7
php_version: 7.4.14 64bit
php_sapi: cgi-fcgi
max_input_variables: 10000
time_limit: 300
memory_limit: 1024M
max_input_time: -1
upload_max_filesize: 256M
php_post_max_size: 256M
curl_version: 7.69.1 OpenSSL/1.0.2k-fips
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 5.7.27-30
client_version: mysqlnd 7.4.14

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/u0304268/data/www/fujimifoto.ru/wp-content
WP_PLUGIN_DIR: /var/www/u0304268/data/www/fujimifoto.ru/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 1024M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8mb4
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips mod_fcgid/2.3.9 Phusion_Passenger/5.3.7
pods-user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36
pods-session-save-path: /var/www/u0304268/data/bin-tmp/
pods-session-save-path-exists: Да
pods-session-save-path-writable: Да
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: Нет
pods-opcode-cache-memcached: Нет
pods-opcode-cache-opcache: Да
pods-opcode-cache-redis: Нет
pods-object-cache-apc: Нет
pods-object-cache-apcu: Нет
pods-object-cache-memcache: Да
pods-object-cache-memcached: Да
pods-object-cache-redis: Нет
pods-memory-current-usage: 22,114M
pods-memory-current-usage-real: 10,000M
pods-network-wide: Нет
pods-install-location: /wp-content/plugins/pods/
pods-developer: Нет
pods-tableless-mode: Нет
pods-relationship-table-enabled: Да
pods-light-mode: Нет
pods-strict: Нет
pods-allow-deprecated: Да
pods-api-cache: Да
pods-shortcode-allow-evaluate-tags: Нет
pods-sessions: Disable sessions
pods-can-use-sessions: Да

`
```

**Comment by professorseo** [comment]
Just FYI that this is the same problem as I am facing.

**Comment by sc0ttkclark** [comment]
Fixed via 7ccee6cadcfe5c524fd51937ce5c2655b7d41b7d
